frontend/packages/knative-plugin/src/actions/providers.ts

```diff
@@ -96,15 +99,15 @@ export const useKnativeServiceActionsProvider = (resource: K8sResourceKind) => {
       DeploymentActionFactory.EditResourceLimits(kindObj, resource),
       CommonActionFactory.ModifyLabels(kindObj, resource),
       CommonActionFactory.ModifyAnnotations(kindObj, resource),
-      CommonActionFactory.Edit(kindObj, resource),
+      editKnativeServiceResource(kindObj, resource, serviceTypeValue),
       ...(resource.metadata.annotations?.['openshift.io/generated-by'] === 'OpenShiftWebConsole'
-        ? [DeleteResourceAction(kindObj, resource)]
+        ? [deleteKnativeServiceResource(kindObj, resource, serviceTypeValue)]
         : [CommonActionFactory.Delete(kindObj, resource)]),
```

Comment:
When the generated-by annotation isn't set, which means the resource is created with the CLI, it shows now

* Edit Function
* Delete Service

Can you also "copy" the `CommonActionFactory.Delete` action so that it calls the origin action but shows "Delete Function"?

Reply:
Updated the PR. PTAL.